### DISCUSSION TITLE
it is possible to download ...

### DISCUSSION THREAD

**Discussion by Alidadm** [leokhoa/laragon#682] [General]
I'm looking for an old version of Laravel 9 instead of Laravel 10, is possible to download Laragon software with Laravel 9 instead of Laravel 10?  
If so, kindly please provide me a URL for me to download it!
thanks. 
AM

**Comment by Pen-y-Fan** [comment]
Yes, you can install version 9 yourself from the command line or add a line to Laravel > Quick App > Configuration.

Copy the existing Laravel=composer create-project laravel/laravel %s --prefer-dist

Paste a copy under the existing one and amend to Laravel9= and after laravel/laravel add :^9, it should look like this:

```text
Laravel9=composer create-project laravel/laravel:^9 %s --prefer-dist
```

Then create your app Laragon menu > Quick app > Laravel9, give it a name and it will be created in the www directory.

<img width="504" alt="laravel9" src="https://github.com/leokhoa/laragon/assets/40126936/0282f9c1-d5e2-441b-887b-f61270da465c">

**Reply by Alidadm** [reply]
Many many thanks so much, it works great. you saved my time and problems I was almost going to uninstall and install it... but glad that got your response right away. 
Again many thanks. 
AM
